package.json

```diff
@@ -37,17 +37,17 @@
 	"license": "MIT",
 	"peerDependencies": {
 		"@agility/content-fetch": "^2.0.0",
-		"next": "^14.0.1"
+		"next": "^15.0.3"
```

Comment:
I opened this PR so that the starters wouldn't throw warnings `@agility/nextjs@14.0.1" has incorrect peer dependency "next@^14.0.1`